src/Symfony/Component/DependencyInjection/Tests/Compiler/FactoryReturnTypePassTest.php

```diff
@@ -44,13 +45,8 @@ public function testProcess()
         $pass = new FactoryReturnTypePass();
         $pass->process($container);
 
-        if (method_exists(\ReflectionMethod::class, 'getReturnType')) {
```

Comment:
Seeing this change it looks like the test was expected to pass on older PHP versions too. And since the test doesn't fail always we should probably rather investigate why they sometimes fail and why they pass under other circumstances.

Reply:
Yes there is an if/else clause in the 3.4 branch, however a few lines above that, there is reference to **FactoryDummy** fixture, that causes the syntax (parse) error, because it does not have a valid PHP 5 syntax:
https://github.com/symfony/symfony/blob/aca3d2c90d3ad01687771ce75602fed5b12bcf2b/src/Symfony/Component/DependencyInjection/Tests/Compiler/FactoryReturnTypePassTest.php#L34
AppVeyor CI test also points out the same issue:
https://ci.appveyor.com/project/fabpot/symfony/builds/21760919?fullLog=true#L2048
Anyway, what can I do to help solve this issue? If there is an issue, to begin with ...